app/src/main/java/org/gdg/frisbee/android/fragment/GdlListFragment.java

```diff
@@ -98,28 +85,25 @@ public void onActivityCreated(Bundle savedInstanceState) {
 
         mClient = new GoogleDevelopersLive();
 
-        mAdapter = new GdlAdapter(getActivity(), ((GdgActivity)getActivity()).getGoogleApiClient());
+        mAdapter = new GdlAdapter(getActivity(), ((GdgActivity) getActivity()).getGoogleApiClient());
         setListAdapter(mAdapter);
 
         setIsLoading(true);
 
         fetchContent(false);
 
-        if(getArguments().getBoolean("active")) {
+        if (getArguments().getBoolean("active")) {

```

Comment:
Should that be part of this PR? I didn't change that code. The only reason why it appears in the diff is because of the reformatting. I'm going to do some clean up in that class anyway so I believe it should go into a different PR.